Proceedings.md

```diff
@@ -30,3 +34,8 @@
   
 - Yáñez, M. A. (2023). _Developing a Quantum Step-sequencer_. **PDF/DOI: [10.5281/zenodo.10206500](https://doi.org/10.5281/zenodo.10206500)** <citebutton-component indexkey="ISQCMC_yanez"></citebutton-component>
 
+### 2021 - 1st ISQCMC - Paper Proceedings
+
+Paper Proceedings for this edition are not available. However, please note that several papers from this edition have evolved into book chapters in:
```

Comment:
It does make sense to specify it here, I agree. 

Do you think we should also mention the new book and link it to 2023?